lib/client/redirect.go

```diff
@@ -107,18 +111,31 @@ func NewRedirector(ctx context.Context, login SSHLoginSSO, config *RedirectorCon
 		return nil, trace.Wrap(err)
 	}
 
+	var callbackAddr string
+	if login.CallbackAddr != "" {
+		callbackURL, err := apiutils.ParseURL(login.CallbackAddr)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		if callbackURL.Scheme == "" {
+			callbackURL.Scheme = "http"
```

Comment:
I believe this should be `https`.  In fact I believe we should enforce the schema to be `https` otherwise we open our customers up to potential MITM attacks.